graphics.cpp

```diff
@@ -0,0 +1,51 @@
+#include "graphics.h"
+
+
+bool Graphics::InitSdl() {
+  if (SDL_Init(SDL_INIT_EVERYTHING) < 0) {
+    std::cout << "SDL Failed to initialize. Error: " << SDL_GetError()
+              << std::endl;
+    return false; //failed to initialize
+  }
+  return true; //successfully initialized
+ }
+
+
+int Graphics::createWindow() {
+  // If SDL can't initialize, exit
+  if (!InitSdl()) {
+    return 1;
+  }
+
+  SDL_Window* window;
+  SDL_Renderer* renderer;
+
+  // SDL method to create our window
+  SDL_CreateWindowAndRenderer(kWindowWidth, kWindowHeight, 0, &window,
+                              &renderer);
+  SDL_SetWindowTitle(window, kGameTitle);
+
+  // Changes background to blue
+  SDL_SetRenderDrawColor(renderer, BackgroundColor.red, BackgroundColor.green,
+                         BackgroundColor.blue, BackgroundColor.alpha);
+
+  // Clear the screen, putting our background color in every pixel
+  SDL_RenderClear(renderer);
+
+  // displays our screen in the window
```

Comment:
Try to be consistent with comment style. How bout:

// Comments are written like sentences. They start with a capital letter and end with a period.